test/unit/spec/MfaVerify_spec.js

```diff
@@ -4015,6 +4018,80 @@ function (Okta,
       });
     });
 
+    Expect.describe('Custom HOTP Factor', function () {
+      testHOTPFactor(setupHOTP, 'testStateToken');
+    });
+
+    Expect.describe('Custom HOTP Factor in IDX', function () {
+      testHOTPFactor(setupHOTPWithIdx, '01bfpkAkRyqUZQAe3IzERUqZGOfvYhX83QYCQIDnKZ');
+    });
+
+    function testHOTPFactor (setupFn, expectedStateToken) {
+      itp('is totp', function () {
```

Comment:
nit: `is TotpForm` (just so it doesnt read `Custom HOTP Factor is totp`, since that would be a little confusing...